espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/exceptions/MoveTargetOutOfBoundsException.java

```diff
@@ -7,8 +7,12 @@
 
     public MoveTargetOutOfBoundsException(long targetX, long targetY, final Rect boundingRect) {
         super(String.format(
-            "The target [%s, %s] for pointer interaction is not in the viewport [%s, %s] and cannot be brought into the viewport",
+            "The target [%s, %s] for pointer interaction is not in the viewport [%s] and cannot be brought into the viewport",
```

Comment:
square brackets are probably also redundant, since the output of the toShortString looks like `[left,top][width,height]`